app/routes/account.rb

```diff
@@ -34,6 +34,30 @@ class Account < Core
         current_user.reset_key
         redirect "/account"
       end
+
+      post '/account/track-mentor/invite' do
```

Comment:
Yah, sorry about that. This project is so out of hand--if I ever get the actual usability worked out I see a huge refactoring in my future.